server/authz/policy.rego

```diff
@@ -124,13 +127,27 @@ allow {
   action == [read, write, write_role, change_password][_]
 }
 
+# Global gitops can write users and modify their roles.
+allow {
+  object.type == "user"
+  subject.global_role == gitops
+  action == [write, write_role][_]
+}
+
 # Team admins can perform all operations on the team users (except changing their password).
 allow {
   object.type == "user"
   team_role(subject, object.teams[_].id) == admin
   action == [read, write, write_role][_]
 }
 
+# Team gitops can write team users and modify their roles.
```

Comment:
If you need the ability to modify a user, don't you need the ability to read it first?